pkg/reconciler/reconciler.go

```diff
@@ -271,6 +273,17 @@ func Run(opts Options) {
 		}
 	}
 
+	// Build an event producer to generate events for parser.Run.
+	scheduler := &events.MetaScheduler{
+		Clock:              parseOpts.Clock,
+		SyncPeriod:         opts.PollingPeriod,
+		FullSyncPeriod:     opts.ResyncPeriod,
+		StatusUpdatePeriod: opts.StatusUpdatePeriod,
+		// TODO: Shouldn't this use opts.RetryPeriod as the initial duration?
```

Comment:
This seems like an existing bug. The Backoff wasn't being built with the RetryPeriod option.